polymetis/polymetis/python/torchcontrol/planning/min_jerk.py

```diff
@@ -197,6 +199,7 @@ def generate_cartesian_target_joint_min_jerk(
     """
     steps = _compute_num_steps(time_to_go, hz)
     dt = 1.0 / hz
+    home_pose = torch.zeros_like(joint_pos_start) if home_pose is None else home_pose
```

Comment:
Are we sure we want the default `home_pose` to be the Franka pointed straight up, instead of e.g. the user-defined rest pose? (I understand this is what we were assuming before, so nbd if it's more annoying to fix)

Reply:
Here I just wanted to have a sensible, platform-agnostic default within the torchcontrol module.
When calling it from RobotInterface I would always pass in the rest_pose defined within the robot model config.